mods/cnc/rules/civilian.yaml

```diff
@@ -430,7 +430,7 @@ C10:
 VICE:
 	Inherits: ^Viceroid
 	AttackWander:
-		WanderMoveRadius: 2
-		MinMoveDelayInTicks: 25
-		MaxMoveDelayInTicks: 45
+		WanderMoveRadius: 1
+		MinMoveDelayInTicks: 1500
```

Comment:
This looks excessive (visceroids will now move only once cell per minute!), and isn't mentioned in your changelog.